spec/models/adviser_spec.rb

```diff
@@ -186,19 +171,19 @@ def save_with_commit_callback(model)
     end
 
     context 'when the firm has changed' do
-      it 'triggers reindexing of the adviser and new firm' do
-        expect(GeocodeAdviserJob).to receive(:perform_later).with(subject)
+      it 'triggers reindexing of the original firm the first time' do
         subject.firm = receiving_firm
         save_with_commit_callback(subject)
+        expect(queue_contains_a_job_for(IndexFirmJob)).to be_truthy
```

Comment:
Does `queue_contains_a_job_for` exist elsewhere? Is it different to `aqueue_contains_a_job_for`?